src/ssh/setup.py

```diff
@@ -7,7 +7,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "0.1.8"
+VERSION = "1.0.0"
 
 CLASSIFIERS = [
     'Development Status :: 4 - Beta',
```

Comment:
I think we should change this. With GA, we no longer should say Beta

Reply:
Surprised to see the License type is MIT license instead of Microsoft public license (Ms-PL).
Is it a common norm for Az cli to use MIT license?